src/global_logic.js

```diff
@@ -601,8 +601,8 @@ module.exports.calcBasedOneSummon = function (summonind, prof, buff, totals) {
             var criticalRatio = 1.0
         }
         //Apply Elemental Resistance if not superior element.
-        damageUP -= totals[key]["typeBonus"] == 1.5 ? 0 : parseFloat(prof.enemyResistance);
-        
+        damageUP -= totals[key]["typeBonus"] == 1.5 ? 0 : Math.max(1, 0.01 * parseFloat(prof.enemyResistance));
```

Comment:
can you check here, I am just reading code now.
- [ ] Math.max is for limit **minimum** value, is it always 1? (not tested)
- [ ] any cases enemyResistance is over the range
  - [x] it's limit min=0 max=100 in FormControl
  - [ ] load from save data may have illegal value

Reply:
it should've been Math.min.
i'll add both sides of acceptable range limit for it here too